apps/ledger-live-desktop/tests/post-playwright-run-sanity-tests.js

```diff
@@ -0,0 +1,33 @@
+// This tests that no "Ledger Live Internal" thread have leaked in the system
+
+const ps = require("ps-node");
+
+function checkProcesses() {
+  ps.lookup({}, function(err, list) {
+    if (err) {
+      throw new Error(err);
+    }
+    const processes = list.filter(
```

Comment:
Just to go further... is there any other process that could be running in the background ?

Reply:
main<>window processes are pretty well bounded (and tested) in electron, but only that one is the extra one we created.